src/Symfony/Bundle/WebProfilerBundle/Resources/config/routing/profiler.xml

```diff
@@ -48,7 +48,7 @@
         <default key="_controller">web_profiler.controller.exception_panel::body</default>
     </route>
 
-    <route id="_profiler_exception_css" path="/{token}/exception.css">
+    <route id="_profiler_exception_css" path="/{token}/exception-css">
```

Comment:
`/{token}/exception` is already used by the previous route.

Change as suggested here:

- https://github.com/symfony/symfony/issues/59045#issuecomment-2547409738

Reply:
I don't think anyone complained about this one for now. And, as you said, this route is called and inlined in the final view, so .. maybe let's keep it that way ?